### PR TITLE
[metrics] improve PSNR

### PR DESCRIPTION
**Pull request by ydcjeff**
Fixes https://github.com/pytorch/ignite/pull/1570#issuecomment-770159407

Description:
- `data_range` became required argument
- add tests for Y channel

Check list:
* [x] New tests are added (if a new feature is added)
* [x] New doc strings: description and/or example code are in RST format
* [x] Documentation is updated (if required)

cc: @vfdev-5 @lidq92 @sdesrozis

### CODE REVIEW

**Review by vfdev-5 — APPROVED** [review]
Looks good to me ! Thanks @ydcjeff !

**Comment by vfdev-5** [comment]
@lidq92 could you also please review this PR

**Comment by sdesrozis** [comment]
@ydcjeff LGTM ! Thank you !

**Comment by lidq92** [comment]
> @lidq92 could you also please review this PR

@ydcjeff LGTM! Thanks!

**Comment by vfdev-5** [comment]
Thanks everyone for the review ! Will merge it once CI is done

File: tests/ignite/metrics/test_psnr.py [hunk 1]
```diff
@@ -56,128 +49,173 @@ def _test_psnr(y_pred, y, data_range, device):
...
+    PSNR(data_range=data_range, output_transform=output_transform, device=metric_device).attach(engine, "psnr")
 
-        data = list(range(n_iters))
-        engine.run(data=data, max_epochs=1)
-        result = engine.state.metrics["psnr"]
-        assert "psnr" in engine.state.metrics
+    data = list(range(n_iters))
+    engine.run(data=data, max_epochs=1)
+    result = engine.state.metrics["psnr"]
+    assert "psnr" in engine.state.metrics
```
Comment (ydcjeff) [inline tests/ignite/metrics/test_psnr.py:109]:
```suggestion
    assert result > 0.0
    assert "psnr" in engine.state.metrics
```

**Pull request merged**